test.js

```diff
@@ -7,12 +7,12 @@ var File = require('vinyl');
 var fs = require('graceful-fs');
 var ghPages = require('./');
 var git = require('./lib/git');
-var github = require('octonode');
-var logSymbols = require('log-symbols');
+var gift = require('gift');
```

Comment:
Can you move this line right under `var ghPages`? I want to keep these `var` declarations in alphabetical order.